src/main/java/uk/gov/objecthash/Util.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.objecthash;
```

Comment:
I think maybe this could be moved into its own `util` package.

Reply:
I think I'd prefer to keep it in this package since it's package private at the moment